js/app.js

```diff
@@ -56,13 +57,62 @@ function App() {
 	 *
 	 * @param {Function} callback Function to be called after loading the config
 	 */
-	function loadConfig(callback) {
+	async function loadConfig(callback) {
```

Comment:
I would like to see the loadConfig being asynced and also being freed from the callback. Have you tried that?